// @file
// @brief Specifies the API version number
//
// ----------------------------------------------------------------
//  Copyright (c) 2016, 2024 by cisco Systems, Inc.
//  All rights reserved.
// -----------------------------------------------------------------
//
//

syntax = "proto3";
package service_layer;
option go_package="github.com/Cisco-service-layer/service-layer-objmodel/grpc/protos;service_layer";

// @addtogroup SLVersion
// @ingroup Common
// @{
// Version Major.Minor.Subversion define the API's version.

// Service Layer API version.
// This is used in the Global init message exchange to handshake client/server
// Version numbers.
enum SLVersion {
    option allow_alias = true;
    SL_VERSION_UNUSED = 0;
    SL_MAJOR_VERSION = 0;
    SL_MINOR_VERSION = 10;
    SL_SUB_VERSION = 0;
}

// @}
